tests/jobs/test_local_task_job.py

```diff
@@ -374,6 +374,29 @@ def test_localtaskjob_double_trigger(self):
 
         session.close()
 
+    @patch.object(StandardTaskRunner, 'return_code')
+    @mock.patch('airflow.jobs.scheduler_job.Stats.incr')
+    def test_raw_task_return_code_metric(self, mock_stats_incr, mock_return_code, create_dummy_dag):
+
+        _, task = create_dummy_dag('test_localtaskjob_double_trigger')
+        mock_stats_incr.reset_mock()
+
+        ti_run = TaskInstance(task=task, execution_date=DEFAULT_DATE)
+        ti_run.refresh_from_db()
+        job1 = LocalTaskJob(task_instance=ti_run, executor=SequentialExecutor())
+
+        mock_return_code.side_effect = [None, -9, None]
+
+        with timeout(10):
+            job1.run()
+
+        mock_stats_incr.assert_has_calls(
+            [
+                mock.call('ti.raw_task_return_code.test_localtaskjob_double_trigger.op1.-9'),
+            ],
+            any_order=True,
```

Comment:
You set "any_order=True", but only one call is expected, looks strange?
Maybe good idea to actually have multiple calls expected from different tasks.

Reply:
Removed any_order. Since return_code is mocked, it it better not to introduce multiple task instances.